dqc_us_rules/dqc_us_0004.py

```diff
@@ -56,21 +81,34 @@ def _min_dec_valid(min_dec):
 
     return min_dec is not None and not isnan(min_dec)
 
+
 def _values_unequal(val1, val2, dec_scale, margin_scale=2):
     """
     Checks the values for equality based on their scaling.
     Returns False if the values are equal, otherwise True.
+
+    :param val1: first value to round
+    :type val1: val
+    :param val2: second value to round
+    :type val2: val
+    :param dec_scale: precision on rounded value
+    :type dec_scale: decimal
+    :param margin_scale: margin of scale for the margin of error
+    :type margin_scale: float
+    :rtype: bool
+    :return: True if the values are not equal
```

Comment:
SG: as convention, it goes `:return:` then ':rtype:` to mimic the parameter block and for ease in sphinx generation